### DISCUSSION TITLE
elytron-security-jdbc + hibernate-reactive-panache

### DISCUSSION THREAD

**Discussion by AntoninRoy** [quarkusio/quarkus#30441] [Q&A]
Hi, I try to configure an app with elytron-security-jdbc and hibernate-reactive-panache.
I have difficulties because they don't use the same drivers (reactive and not reactive) 

**My first question is to know if Quarkus allow us to do that ?** 

**If yes, does anybody know how to solve this error  ?**  

`ERROR [io.qua.run.Application] (Quarkus Main Thread) Failed to start application (with profile dev): java.lang.IllegalStateException: Booting an Hibernate Reactive serviceregistry on a non-reactive RecordedState!`

Here is my application.properties : 

```
quarkus.datasource.reactive = true
quarkus.datasource.db-kind = postgresql
quarkus.datasource.username = ***
quarkus.datasource.password = ***
quarkus.datasource.jdbc.url = jdbc:postgresql://***
quarkus.datasource.reactive.url = postgres://***


quarkus.security.jdbc.enabled=true
quarkus.security.jdbc.principal-query.sql=SELECT u.password, u.role FROM test_user u WHERE u.username=? 
quarkus.security.jdbc.principal-query.clear-password-mapper.enabled=true 
quarkus.security.jdbc.principal-query.clear-password-mapper.password-index=1
quarkus.security.jdbc.principal-query.attribute-mappings.0.index=2 
quarkus.security.jdbc.principal-query.attribute-mappings.0.to=groups

quarkus.hibernate-orm.database.generation = drop-and-create
```

[a comment by an automated account is not shown]

**Comment by sberyozkin** [comment]
Reactive equivalents of `security-elytron-jdbc` and `security-elytron-jpa` are most likely needed

**Reply by AntoninRoy** [reply]
Do you know if they already exist ? of if i can follow a topic to be notify when someone will work on ?

**Reply by sberyozkin** [reply]
They don't yet unfortunately

**Reply by AntoninRoy** [reply]
ok, thanks
